### ISSUE TITLE
LayoutLMV2 model fine tuning options 

### ISSUE THREAD

**Issue by khadkechetan** [microsoft/unilm#434]
Hi,
I have been working on Invoice data extraction using the LayoutLMV2 model. I have been getting a good results with limited amount of data. 
I am facing issue with model finetuning. Could u please help me, how can I finetune different hyperparameters of LayoutLMv2 model. It would be great if you could share any link of article for reference. 

Thanks in advance.

**Comment by wolfshow** [comment]
@khadkechetan You may try this https://huggingface.co/nielsr/layoutlmv2-finetuned-funsd

**Comment by khadkechetan** [comment]
Thanks for the prompt reply. 
I can see only two hyperparameters learning rate and optimizer. Does it have more hyper parameters?

**Comment by NielsRogge** [comment]
Hi @khadkechetan,

this is the notebook that I used to get that checkpoint: https://github.com/NielsRogge/Transformers-Tutorials/blob/master/LayoutLMv2/FUNSD/Fine_tuning_LayoutLMv2ForTokenClassification_on_FUNSD_using_HuggingFace_Trainer.ipynb

**Comment by ranpox** [comment]
For more hyper params, please refer to https://huggingface.co/transformers/v4.5.1/main_classes/trainer.html#transformers.TrainingArguments.
